app/views/api/v2/key_performance_indicators/assessment_status.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+json.data({
```

Comment:
Again I think this will be vastly simplified after we make this RESTful. Really I bet we can replace all of these files with one file that is something like:
```
# frozen_string_literal: true

json.data(@kpi.values)
```